services/user.js

```diff
@@ -251,6 +252,15 @@ class User {
         if (params.status && Object.values(USER.STATUSES).includes(params.status)) {
             updatedUser.userStatus = params.status;
         }
+        if ((updatedUser.role && params.role === USER.ROLES.DC_POC) || (!updatedUser.role && user[0]?.role === USER.ROLES.DC_POC)) {
+            if (!params.dataCommons || params.dataCommons.length === 0) {
```

Comment:
We should allow client to omit dataCommons parameter, if current role is DC POC and dataCommons has values in the DB.

Reply:
Updated to allow for this scenario.